openwisp_users/multitenancy.py

```diff
@@ -8,7 +8,34 @@
 OrganizationUser = load_model('openwisp_users', 'OrganizationUser')
 
 
-class MultitenantAdminMixin(object):
+class SelectOrgMixin(object):
+    def select_organization(self, request, form):
+        '''
+        When the user has access to only one organization
+        automatically set that org as the default value
+        '''
+        if request.method == 'POST':
+            return
+        fields = form.base_fields
+        if 'organization' in fields:
+            org_field = fields['organization']
+            org_field.initial = request.user.selected_org(org_field)
+            org_field.empty_label = self._get_empty_level(
+                request.user.is_superuser, org_field
+            )
+
+    def _get_empty_level(self, superuser, org_field):
```

Comment:
@niteshsinha17 it should be _get_empty_label

Reply:
Oh! Thanks @devkapilbansal